.github/workflows/ci.yml

```diff
@@ -61,7 +68,7 @@ jobs:
 
       - name: Run Molecule tests
         run: |
-          molecule test
+          molecule test -s "${{ matrix.scenario }}"
         env:
```

Comment:
Using `matrix` functionality, so both (independent) scenarios can run in parallel